.github/workflows/triage-move-labelled.yml

```diff
@@ -0,0 +1,313 @@
+name: Move labelled issues to correct boards and columns
```

Comment:
All triage files are specific to Element.
So you should remove them from the .github/workflows folder.

Reply:
Done.